login_test.py

```diff
@@ -30,4 +30,40 @@ def test_unverifiedUserFirstLogin(driver, unverified_user_via_medlemshjaelp):
     assert get_form_field_value(driver, form_id="input_username") == user["email"]
 
 
+# Test specification number 2
+def test_unverifiedUserFirstLoginDuplicateTab(driver, unverified_user_via_medlemshjaelp):
+    user = unverified_user_via_medlemshjaelp
+
+    navigate_to_page("login", driver)
+    first_tab = driver.current_window_handle
+
+    navigate_to_page("login", driver, new_tab=True)
+    WebDriverWait(driver, 10).until(EC.number_of_windows_to_be(2))
+    second_tab = [x for x in driver.window_handles if x != first_tab][0]

```

Comment:
And `navigate_to_page` could return the current handle automatically.

Reply:
Nice!